lib/marks.rb

```diff
@@ -1,4 +1,5 @@
 module Mark
   ROUND = "O"
   CROSS = "X"
+  EMPTY = " "
```

Comment:
I agree, it seems a bit weird. I removed it as I need the id for the empty spot.